test/integration/image-optimizer/test/index.test.js

```diff
@@ -495,9 +578,12 @@ function runTests({ w, isDev, domains = [], ttl, isSharp }) {
       `public, max-age=0, must-revalidate`
     )
     expect(res2.headers.get('Vary')).toBe('Accept')
+    expect(res2.headers.get('Content-Disposition')).toEqual(
```

Comment:
I don't think we need to set the header in the case of a 304

https://developer.mozilla.org/en-US/docs/Web/HTTP/Status/304

Reply:
yeah, makes sense 👍